website/docs/monorepo.md

```diff
@@ -0,0 +1,97 @@
+---
```

Comment:
Monorepo is a pretty niche use case so I don't think we need that much explanation on how to use docs with a monorepo structure - we might not even need this page at all.

We want to keep our docs high signal for the common use cases.

The most we should do is to add a small section under the docs section to mention how monorepos can leverage the `exclude` option.

If we really want this page, it should be under docs category and not at the root level.

You're welcome to write a blog post on your private blog to elaborate on using Docusaurus for monorepo docs but I don't think we want to include it in the core docs.